code/datums/status_effects/buffs.dm

```diff
@@ -823,6 +829,7 @@
 	. = ..()
 
 /datum/status_effect/bookwyrm
+	id = "bookworm"
```

Comment:
im on the fence if this should be worm, or wyrm. however it should probably be the same as the datum
```suggestion
/datum/status_effect/bookwyrm
	id = "bookwyrm"
```